modules/cnv_storage_wizard_fields_web.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//cnv_storage_wizard_fields_web.adoc
+// * cnv_users_guide/cnv_users_guide.adoc
+
+[[cnv-storage-wizard-fields-web]]
+== Storage fields
+|===
+|Name | Description
+
+|Create Disk
+|Create a new disk for the virtual machine.
+
+|Attach Disk
+|Select an existing disk to attach to the virtual machine.
+
+|DISK NAME
+|Name of the disk.
+
+|SIZE (GB)
+|Size, in GB, of the disk. 
+
+|STORAGE CLASS
+|`StorageClass` name for the underlying PVC. 
```

Comment:
actually it is really the name of the underlying storage class. Storage class defines where the storage will be living + some characteristics of the storage.